tool/client.c

```diff
@@ -666,7 +666,7 @@ int report_results_client(xtt_identity_type *requested_client_id,
         return 1;
     }
     printf("Server assigned me id: %s\n", my_assigned_id_as_string.data);
-    write_ret = xtt_save_to_file((unsigned char*)my_assigned_id_as_string.data, sizeof(xtt_identity_string), assigned_client_id_out_file);
+    write_ret = xtt_save_cert_to_file((unsigned char*)my_assigned_id_as_string.data, sizeof(xtt_identity_string), assigned_client_id_out_file);
```

Comment:
That sounds good. Should I create a general permissions for those two cases and when we read in from the nvram? I think it would be the same privilege level as a certificate, so probably not _super_ necessary, but maybe slightly more readable.

Reply:
I think just calling `xtt_save_to_file` with an explicit `0644` passed in for the permissions will be the clearest. No need for a new `#define`